bin/commander.js

```diff
@@ -130,7 +130,9 @@ async function runGitRepositoryCloneStep( abortMessage ) {
 	await runStep( 'Cloning the Git repository', abortMessage, async () => {
 		console.log( '>> Cloning the Git repository' );
 		const simpleGit = SimpleGit();
-		await simpleGit.clone( gitRepoURL, gitWorkingDirectoryPath );
+		await simpleGit.clone( gitRepoURL, gitWorkingDirectoryPath, [
+			'--depth 1',
```

Comment:
In my initial tests, just this change causes the script to fail later, can't remember if it's when tagging the release or when committing to master. We might want to test on a fork.

Reply:
This sounds reasonable, shallow clones are only to a single branch, if I remember correctly bumping the depth to 5 or 10 resolved this type of issue for me with Core Travis CI yeras ago.

Possibly adding `--no-single-branch` is another option, see https://www.git-scm.com/docs/git-clone#Documentation/git-clone.txt---depthltdepthgt